src/test/java/BaseTest.java

```diff
@@ -33,44 +30,38 @@ public static void chromeConfigs() {
         }
     }
 
-    @BeforeMethod
+    @BeforeClass
     @Parameters({"BaseURL"})
     public void launchBrowser(String BaseURL) {
         ChromeOptions chromeOptions = new ChromeOptions();
         chromeOptions.addArguments("--disable-notifications");
 
         driver = new ChromeDriver(chromeOptions);
         driver.manage().window().maximize();
-        actions = new Actions(driver);
-        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
         url = BaseURL;
         driver.get(url);
     }
 
-   // @BeforeTest
-   // public void login() {
-       // By emailLocator = By.xpath("//input[@placeholder='Email Address']");
-        //wait.until(ExpectedConditions.elementToBeClickable(emailLocator));
-       // WebElement emailField = driver.findElement(emailLocator);
-       // emailField.click();
-       // emailField.sendKeys("holostenco.yuliya@gmail.com");
 
-       // By passwordLocator = By.xpath("//input[@type='password']");
-       // wait.until(ExpectedConditions.elementToBeClickable(passwordLocator));
-       // WebElement passwordField = driver.findElement(passwordLocator);
-        //passwordField.click();
-        //passwordField.sendKeys("te$t$tudent");
 
-       // By submitButtonLocator = By.xpath("//button[@type='submit']");
-       // wait.until(ExpectedConditions.elementToBeClickable(submitButtonLocator));
-       // driver.findElement(submitButtonLocator).click();
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.